src/main/java/com/stripe/model/Dispute.java

```diff
@@ -138,6 +139,14 @@ public void setStatus(String status) {
 		this.status = status;
 	}
 
+	public String getNetworkReasonCode() {
```

Comment:
Mind also commenting this method? I think it will be slightly better because it's more likely to show up in a user's IDE if they happen to be using one (which, in Java, there's a reasonable chance).

Otherwise, LGTM. Thanks!